torch/csrc/jit/codegen/cuda/fusion_segmenter.h

```diff
@@ -271,7 +287,7 @@ class TORCH_CUDA_CU_API SegmentedFusion {
   void draw();
 
   //! Debug print for segmented fusions
-  void print() const;
+  void print();
```

Comment:
Actually it was because the `GroupDependencyAnalysis`, which print uses, relies on a mutable reference, but it didn't need to. I refactored and print is constant again. Thanks for pointing this out 👍